tests/integration_test.rs

```diff
@@ -0,0 +1,35 @@
+use fvm_shared::econ::TokenAmount;
+use ipc_sdk::subnet_id::{ROOTNET_ID};
+use ipc_subnet_actor::{ConsensusType, ConstructParams};
+use ipc_client::lotus::LotusClient;
+use ipc_client::manager::{LotusSubnetManager, SubnetManager};
+use crate::setup::{LOCAL_JSON_RPC_HTTP_URL};
+
+mod setup;
+
+#[tokio::test]
+#[ignore] // ignore this test for now as it's still developing
```

Comment:
Maybe keep it out of the repository for now then?

Reply:
This integration test is working now and it won't be triggered only if explicitly run it. Probably we can keep it for now to be example and use it as some sanity test for checking any updates to `lotus` that might break rust side. Eventually it will be moved to json rpc server, once this happens, I will delete this file. Hope this is ok.

Reply:
What about moving this to a gits file outside this repo and pointing to it from the dev notes you started above? I am just worried that we may forget to remove it and updated in the future leading to dirt in the repo.